src/v2/Apps/Settings/settingsRoutes.tsx

```diff
@@ -97,6 +97,26 @@ const DeleteAccountRoute = loadable(
   }
 )
 
+const SavedSearchAlertsApp = loadable(
+  () =>
+    import(
+      /* webpackChunkName: "savedSearchAlertsAppBundle" */ "./Routes/SavedSearchAlerts/SavedSearchAlertsApp"
```

Comment:
@damassi should we replace `savedSearchAlertsAppBundle` on `settingsBundle`?

Reply:
I think its fine to include everything in `settingsBundle`!